cleverhans/attacks_tf.py

```diff
@@ -1320,3 +1320,153 @@ def deepfool_attack(sess, x, predictions, logits, grads, sample, nb_candidate,
     # need to clip this image into the given range
     adv_x = np.clip((1+overshoot)*r_tot + sample, clip_min, clip_max)
     return adv_x
+
+
+def compute_mask(levels, low, high, clip_min=0., clip_max=1.,
+                 thermometer=False):
+    """Get a mask of allowable perturbations in the interval (low, high).
+
+    For example, assume that we uniformly discretize the values
+    between 0 and 1 into 10 bins each represented by either a one hot
+    encoding or a thermometer encoding. Then compute_mask(10, .3, .7)
+    would return [0., 0., 0., 1., 1., 1., 1., 0., 0., 0.]. Note that it's
+    output is independent of the encoding used.
+
+    Args:
```

Comment:
The docstring format doesn't look consistent across the different functions, is that intentional?